spec/javascripts/collections/files_collection_spec.js

```diff
@@ -36,6 +37,19 @@ describe('FileCollection', function() {
     });
   });
 
+  describe('.createForFileType', function() {
+    it('passes fileType to files', function() {
+      var fileType = new pageflow.FileType({collectionName: 'image_files',
+                                            model: Model,
+                                            matchUpload: /^image/});
```

Comment:
There is now a factory to create a fixture image file type. See other specs in this file.